Druid/DRBytecodeIRGenerator.class.st

```diff
@@ -289,6 +289,20 @@ DRBytecodeIRGenerator >> push: aValue [
 	^ self popOperand
 ]
 
+{ #category : #'stack management' }
+DRBytecodeIRGenerator >> pushFrameForCode: aDRCode receiver: aReceiver [
+
+	| frame |
+	frame := super pushFrameForCode: aDRCode receiver: aReceiver.
+
+	"If this is the base frame, it should continue with the next bytecode at the end"
+	executionState baseFrame = frame ifTrue: [
+		frame exitBasicBlock endInstruction:
+			(self instantiateNoResultInstruction: DRContinueNextBytecode) ].
+
+	^ frame
+]
```

Comment:
Also exists the method `finishCodeInterpretation: lastFrame` that is called when the last frame is popped (I don't know if it's the same to add the instruction on push or pop)

Reply:
Yeah, I don't know if adding it at the end or at the beginning of the frame...